lib/matplotlib/bezier.py

```diff
@@ -316,7 +317,7 @@ def _f(xy):
 
 def get_cos_sin(x0, y0, x1, y1):
     dx, dy = x1 - x0, y1 - y0
-    d = (dx * dx + dy * dy) ** .5
+    d = (dx * dx + dy * dy) ** .5 or 1  # Account for divide by zero
```

Comment:
As a side note, my negative reaction to it reflected my earlier days working with Perl--it's a typical Perl-style construct, compact but a bit tricky, relying on evaluating an expression in two distinct ways (numerical and Boolean) in a single statement.